application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -565,6 +565,27 @@ define('diagram-link-editor', [
       a.nodeName.toLowerCase() != b.toLowerCase() ? !1 : null == c || a.getAttribute(c) == d;
   };
 
+  var originalShowBackgroundImageDialog = EditorUi.prototype.showBackgroundImageDialog
```

Comment:
You don't use it. This is helpful in order to not copy-paste the code from drawio, since we would need to maintain it at upgrade. Like this, you would simply call it inside `EditorUi.prototype.showBackgroundImageDialog` and apply what you need on the result. 
But, looking at the method, this is not possible here. SO you don't need this `original..`
Could add in return some notes for what was changed? This would make an upgrade easier